src/when.test.js

```diff
@@ -223,6 +223,16 @@ describe('When', () => {
       expect(fn(2)).toEqual('y')
     })
 
+    it('should handle symbol matchers', () => {
+      const fn = jest.fn()
+      const symbol = Symbol.for(`sym`)
+      when(fn).calledWith(symbol, 2).mockReturnValue('x')
+
+      expect(fn(5)).toBeUndefined()
```

Comment:
Do you want an assertion that is calling the fn and getting 'x' back? Like the happy path.

Reply:
Sure, I just copied the test below it

Reply:
actually that was a good catch as it found another similar bug. See updated code